foreman/dynflow-executor.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
```

Comment:
Is it standard to name the service after the bin script? I'd think `dynflow` would be a simpler service to remember.